.github/workflows/auto-pr-submodules.yml

```diff
@@ -0,0 +1,38 @@
+# This is a basic workflow to help you get started with Actions
+name: Update Submodules and Submit a PR
+
+# Controls when the action will run.
```

Comment:
I don't think this and other similar comments are necessary, because that's available in the workflow docs.